account_banking_make_deposit/demo/account_banking_make_deposit_demo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
```

Comment:
This only deals with `deposit.method`, please rename to `demo/deposit_method.xml`